src/app/composables/storage/EventStorageStrategy.ts

```diff
@@ -0,0 +1,40 @@
+import type { EventVisibility } from '~~/gql/generated/graphql'
+
+export interface EventStorageStrategy {
+  saveEvent(form: EventFormData): Promise<void>
+}
+
+export interface EventFormData {
+  name: string
+  slug: string
+  isInPerson: boolean
+  isRemote: boolean
+  category: string
+  id: string
+  authorAccountId: string
+  description: string
+  end: string
+  inviteeCountMaximum: string
+  location: string
+  start: string
+  url: string
+  visibility: EventVisibility | null
+  street: string
+  houseNumber: string
+  additionalStreet: string
+  postcode: string
+  city: string
+  country: string
+  startDate: string
+  startTime: string
+  endDate: string
+  endTime: string
+  isRecurring: boolean
+  frequency: string
+  recurringEndDate: string
+  address: string
+  website: string
+  images: File[]
+  coverImage: File | null
+  previewUrls: string[]
```

Comment:
Can we sort this alphabetically? This makes it easier to read and check if all properties are covered. If you select all lines you can let vscode sort automatically.